tests/vspec/test_structs/expected-signals-types.yaml

```diff
@@ -0,0 +1,68 @@
+A:
+  description: Branch A.
+  type: branch
+
+A.NestedStructSensor:
+  datatype: VehicleDataTypes.TestBranch1.NestedStruct
+  description: A rich sensor with user-defined data type.
+  type: sensor
+
+A.ParentStructSensor:
+  datatype: VehicleDataTypes.TestBranch1.ParentStruct
+  description: A rich sensor with user-defined data type.
+  type: sensor
+
+A.UInt8:
+  datatype: uint8
+  description: A uint8.
+  type: sensor
+  unit: km
+
+CustomDataTypes:
```

Comment:
I see that you put the type declarations within "CustomDataTypes". I think putting types in a separate branch makes sense as we then avoid the risk of name conflicts (as we have said that having a type called `A.UInt8` is allowed even if you have a signal called `A.UInt8`). But I would like to get feedback from others and we should also better describe this design decision in the markdown files for vspec2x

Reply:
Meeting notes:

- Ulf: Shall we always use "Custom" as prefix?
- Sebastian: Possible "Complex"?
- Decision: Use "ComplexDataTypes"